packages/shared-components/src/components/onto-search-resource-input/onto-search-resource-input.tsx

```diff
@@ -89,11 +89,11 @@ export class OntoSearchResourceInput {
         </div>
         <span class="hint">{TranslationService.translate('rdf_search.labels.hint')}</span>
         <section class="autocomplete-results-wrapper">
-          {this.searchResult?.suggestions.getItems().map((suggestion) => (
-            <p key={suggestion.id}
+          {this.searchResult?.getSuggestions().getItems().map((suggestion) => (
+            <p key={suggestion.getId()}
```

Comment:
It's OK to have some fields open via get method so that you can bind directly to a property. If we decouple the input model (server response) from the internal (UI) model through the mappers, then we can safely open some parts of the UI model. Otherwise we'd sacrifise some performance.